lib/rspec/rails/view_rendering.rb

```diff
@@ -44,6 +44,13 @@ def render_views?
       #
       # @private
       class EmptyTemplateResolver < ::ActionView::FileSystemResolver
+
+        def initialize(path, pattern=nil)
+          unless path.is_a?(::ActionView::Resolver)
+            super
+          end
+        end
+
```

Comment:
This bit is to fix #1551 isn't it? If so adding your custom resolver instance to the view path with result in an instance of `EmptyTemplateResolver` with a `nil` path instance variable instead. I thought the idea would be to put the custom resolver into the path without wrapping it.